components/feature/search/src/main/assets/searchplugins/amazon-nl.xml

```diff
@@ -5,8 +5,11 @@
 <SearchPlugin xmlns="http://www.mozilla.org/2006/browser/search/">
```

Comment:
github won't let me comment in the right place, so I'll comment here.

amazon-nl is in an `experimental-hidden` section at the moment, so I believe it won't be showing up. I think that needs to be moved, and a test added as well as I don't see one for NL.

Reply:
Sure, I wasn't sure the reason why NL was not in the visible section so decided not to move it.  But we can always remove it if there's any user complaint.  I'll move it to the visible section.  Thanks,

Reply:
To be safe, I'll only move the one in locale NL.  Not the one in locale fy-NL.  Thanks,